src/api/controllers/gamesController.ts

```diff
@@ -1,3 +1,3 @@
-export function dice(steps : number) {
-    return Math.floor(Math.random() * steps + 1)
+export function dice(steps: number) {
```

Comment:
```suggestion
export function dice(steps: number): number {
```
Podrías tipar el resultado de la función y así al usarla sabrías el tipo de dato resultante.